pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -148,13 +169,28 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 		pids:                make(map[int32]bool),
 		cipherMetrics:       cipher.NewAgentMetrics(agentName),
 		metricInterval:      10,
+		resourcesUsedByVMs:  make(map[uuid.UUID]ResourcesUsedByVM),
+		hostCpusPinned:      make(map[int]bool),
+		cpusReservedForEve:  2,
```

Comment:
That's interesting.

At the moment, we provide the setting for the EVE cgroups CPU set with kernel arguments.
And you are right, we provide the kernel arg value of 1 now: `eve_max_vcpus=1`.

But if we check the script that setup the corresponding CPU set, we will see the following:
https://github.com/lf-edge/eve/blob/d57dc774a6f2f6c776e95ec8f065709964629c3b/pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup#L109

So, the result is the EVE services use 2 CPUs: 0 and 1:

```
ecc8bf46-f34c-4595-b788-c64306a6a595:~# for i in $(find /sys/fs/cgroup/cpuset/ -type d) ; do echo -n "$i " ; cat $i/cpuset.cpus ; done | grep eve\/ | grep -v "cpuset/ "
/sys/fs/cgroup/cpuset/eve/services 0-1
/sys/fs/cgroup/cpuset/eve/services/eve-edgeview 0-1
/sys/fs/cgroup/cpuset/eve/services/edgeview 0-1
/sys/fs/cgroup/cpuset/eve/services/wwan 0-1
/sys/fs/cgroup/cpuset/eve/services/newlogd 0-1
/sys/fs/cgroup/cpuset/eve/services/xen-tools 0-1
/sys/fs/cgroup/cpuset/eve/services/wlan 0-1
/sys/fs/cgroup/cpuset/eve/services/debug 0-1
/sys/fs/cgroup/cpuset/eve/services/guacd 0-1
/sys/fs/cgroup/cpuset/eve/services/pillar 0-1
/sys/fs/cgroup/cpuset/eve/services/lisp 0-1
/sys/fs/cgroup/cpuset/eve/services/vtpm 0-1
/sys/fs/cgroup/cpuset/eve/services/watchdog 0-1
/sys/fs/cgroup/cpuset/eve/services/sshd 0-1
```
That's why I use 2 at the moment. 

But maybe it would be better to fix the `init.d/010-eve-cgroup` file and parse the corresponding value from the kernel args. 

Should I fix it in the scope of this PR?

Reply:
> So, the result is the EVE services use 2 CPUs: 0 and 1:

But isn't that a bug?
Makes sense fixing that in a separate pre-ceeding PR and not confuse the scope of this PR.

Reply:
So, the plan is:
1) Fix the `init.d/010-eve-cgroup` script
2) Create a PR for the fix and merge it
3) Add a function to read the corresponding value (`eve_max_vcpus`) from the `/proc/cmdline` file
4) Continue with this PR
Right?

Reply:
Makes sense.

Reply:
Done